source/includes/_rars_top_referrers.md

```diff
@@ -0,0 +1,166 @@
+--header 'Authorization: xxx' \
+--header 'x-api-key: xxx'
+```
+
+> Example Response
+
+```json
+{
+    "api_name": "top_referrer",
+    "api_run_date": "2022-11-17",
+    "start_date": "2022-11-10",
+    "end_date": "2022-11-15",
+    "time_zone": "America/Los_Angeles",
+    "interval_size": "calendar_week",
+    "global_master_advertiser_id": "TEST_1",
+    "report_data": {
+        "sources": [
+            {
+                "name": "DIRECT",
+                "master_campaign_id": 1,
+                "is_paid": true,
+                "top_referrers": [
+                    {
+                        "referrer_host": "facebook.com",
+                        "count": 5
+                    }
+                ]
+            },
+            {
+                "name": "DIRECT",
+                "master_campaign_id": 1,
+                "is_paid": true,
+                "top_referrers": [
+                    {
+                        "referrer_host": "fifa.com",
+                        "count": 14
+                    }
+                ]
+            },
+            {
+                "name": "SEARCH",
+                "master_campaign_id": 1,
+                "is_paid": true,
+                "top_referrers": [
+                    {
+                        "referrer_host": "google.com",
+                        "count": 14
+                    }
+                ]
+            },
+            {
+                "name": "SEARCH",
+                "master_campaign_id": -1,
+                "is_paid": false,
+                "top_referrers": [
+                    {
+                        "referrer_host": "fifa.com",
+                        "count": 3
+                    }
+                ]
+            },
+            {
+                "name": "SOCIAL",
+                "master_campaign_id": -1,
+                "is_paid": false,
+                "top_referrers": [
+                    {
+                        "referrer_host": "fifa.com",
+                        "count": 3
+                    }
+                ]
+            },
+            {
+                "name": "DIRECT",
+                "master_campaign_id": -1,
+                "is_paid": false,
+                "top_referrers": [
+                    {
+                        "referrer_host": "fifa.com",
+                        "count": 13
+                    }
+                ]
+            },
+            {
+                "name": "OTHER",
+                "master_campaign_id": -1,
+                "is_paid": false,
+                "top_referrers": [
+                    {
+                        "referrer_host": "fifa.com",
+                        "count": 9
+                    }
+                ]
+            }
+        ]
+    }
+}
+```
+
+The body of the API response is an array of source objects.
```

Comment:
This section is a little unclear. Can you update the headers and rearrange it like this?
```
The main body of the API response is found in the report_data object.
**Base Object**
|Field Name|Datatype|Description|
|---|---|---|
|api_name|String|Name of the API|
|api_run_date|String|Date report was run|
|start_date|String|Start date of report|
|end_date|String|End date of report|
|time_zone|String|Time Zone|
|interval_size|String|Interval Size ('none', 'day', 'calendar_week' or 'calendar_month')|
|report_data|Object|[Report Data Object](#topreferrerreportdata)

<a id="topreferrerreportdata"></a>
**Report Data Object**
Field Name | Datatype | Nullable | Description
---------- | -------- | -------- | -----------
sources | Array | Yes | An array of source objects

**Source Object**
Field Name | Datatype | Nullable | Description
---------- | -------- | -------- | -----------
name | String | Yes | The type of campaign SEARCH, OTHER, DIRECT
master_campaign_id | String | Yes | The master campaign id
is_paid | boolean | No | Is the campaign a paid campaign
top_referrers | Array | No | An array of [top referrer objects](#array-of-top-referrers)

<a id="array-of-top-referrers"></a>
**Array of top referrers**
Field Name | Datatype | Nullable | Description
---------- | -------- | -------- | -----------
referrer_host | String | No | The hostname
count | Integer | No | The count of referrals from the specified host
```